testutil/network/sessionnet/sessions.go

```diff
@@ -0,0 +1,108 @@
+package sessionnet
+
+import (
+	"context"
+	"encoding/base64"
+	"os"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/pokt-network/poktroll/pkg/relayer"
+	"github.com/pokt-network/poktroll/pkg/relayer/session"
+	"github.com/pokt-network/poktroll/testutil/testrelayer"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+// GetSession sends a query using the test CLI to get a session for the inputs provided.
+func (memnet *inMemoryNetworkWithSessions) GetSession(
+	t *testing.T,
+	serviceId string,
+	appAddr string,
+) *sessiontypes.Session {
+	t.Helper()
+	ctx := context.TODO()
+	net := memnet.GetNetwork(t)
+
+	sessionQueryClient := sessiontypes.NewQueryClient(net.Validators[0].ClientCtx)
+	res, err := sessionQueryClient.GetSession(ctx, &sessiontypes.QueryGetSessionRequest{
+		ApplicationAddress: appAddr,
+		Service:            &sharedtypes.Service{Id: serviceId},
+		// NB(#196): intentionally omitting BlockHeight (i.e. session start height)
+		// as it is difficult to predict in the general case.
+		// BlockHeight:
```

Comment:
This is working because nothing is currently enforcing it on-chain right? At some point this would be mandatory (?)

Reply:
This works  because, IIRC, if the block height is 0, it will be set to the correct value during on-chain validation.